QSB/Messaging/MessageType.cs

```diff
@@ -12,7 +12,13 @@ public enum MessageType
         Leave = MsgType.Highest + 6,
         FullState = MsgType.Highest + 7,
         FullStateRequest = MsgType.Highest + 8,
-        Event = MsgType.Highest + 9
+        FlashlightActiveChange = MsgType.Highest + 9,
+        SignalscopeActiveChange = MsgType.Highest + 10,
+        TranslatorActiveChange = MsgType.Highest + 11,
+        ProbeLauncherActiveChange = MsgType.Highest + 12,
+        PlayerJoin = MsgType.Highest + 13,
+        PlayerLeave = MsgType.Highest + 14,
```

Comment:
we've now got Leave and PlayerLeave, Join and PlayerJoin :P

Reply:
yeah, lol